### ISSUE TITLE
filter by city, country, gender, intern, junior_developer etc....

### ISSUE THREAD

**Issue by berlintam** [rubycorns/whatdojuniorsgetpaid#8]
It would be great to have filters on the junior_storyies#index page so that users can filter the stories by attributes that mean the most to them. (think similar to our filter here http://rorganize.it/groups) I think we can start with:

* gender
* country/city
* intern/junior developer
* freelancer/not freelancer
* technology

thoughts @sareg0?

**Comment by sareg0** [comment]
I think that sounds great. It would be a great way for jrs to find the data they're looking for, since it could be a little while between now and when we get any real visualisations/data analysis going on. I think the first focus should be on being able to filter by location.... Seems like the most logical thing people will go to first.

**Comment by TPei** [comment]
We could abuse rails admin for that. By disabling all editing actions (new, edit) and things like history etc...
I'm not sure I like it, since it breaks railsadmin for actual use by administrators, but it would be an instant fix (see #15)
![screen shot 2015-12-07 at 10 18 17](https://cloud.githubusercontent.com/assets/4004343/11623292/43027c48-9ccc-11e5-86c1-5def5924fa59.png)

**Comment by berlintam** [comment]
hmmmm but the idea is that everyone can filter by gender/city etc.... on the index page, so this wouldn't just be used by admins. I don't think that we would need an admin area yet (if ever)....

Implementing basic filtering on the front end is definitely possible, plus then we have more styling options (which is near impossible with rails admin). 

wdyt?

**Comment by TPei** [comment]
Agreed, don't think this is the best idea, would be a lazy quick fix, but cause pain later on. Was just a quick thought this morning.

**Comment by TPei** [comment]
Ok, I'll work on this. Can you assign me @berlintam? Is that even possible when I'm not part of the rubycorns group?

**Comment by berlintam** [comment]
I just looked and it doesn't seem possible. But you are officially assigned in my books :) Thanks so much!

**Comment by TPei** [comment]
Finally managed to find some time to work on this...
Progress can be tracked in PR #16
